src/avatar.h

```diff
@@ -44,6 +45,7 @@ class avatar : public player
         bool create( character_type type, const std::string &tempname = "" );
         void randomize( bool random_scenario, points_left &points, bool play_now = false );
         bool load_template( const std::string &template_name, points_left &points );
+        static void save_template( const avatar &u, const std::string &name, const points_left &points );
```

Comment:
Why is this a static function of `avatar`, taking an object of type `avatar` as parameter? Why not make it non-static and use the implicit `this` pointer:
```suggestion
        void save_template( const std::string &name, const points_left &points ) const;
```